extensions/adobe/experience/campaign/orchestration/experienceevent.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/experience/campaign/orchestration/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey experience event",
+  "type": "object",
+  "description": "Orchestration extension to ExperienceEvent",
+  "definitions": {
+    "journey": {
```

Comment:
There was lot more attributes to add to this journey definition as well. We haven't yet decided on that, but making this "object" will make it easy to expand in future.
That is also one possible option, but whatever you are suggesting makes sense from the run time aspect of the system (journey_instance is going to hold the version), but to store static definition of journey version (which has the state machine) itself, it will make more sense to split them into journey and journey version. Fundamentally as well we have very clear definition of journey and jourenyVersion which are linked together. So creating a new journeyVersion definition will be useful as it will be referenced again.